src/bot.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, screen: Screen, pick_corpose_on_start: bool = False):
         self._pausing = False
         self._current_threads = []
         self._no_stash_counter = 0
+        self.out_of_gold = False
+        self.wait_to_repair = 0
+
```

Comment:
```suggestion
        self.wait_to_repair = 0
```
Remove extra new line